### DISCUSSION TITLE
Rdio Scanner + RTLSDR Airband problem

### DISCUSSION THREAD

**Discussion by poncheck** [chuot/rdio-scanner#117] [General]
Hello!

I am a newbie user of Rdio Scanner + RTLSDR Airband and I got a problem with Talkgroups and Systems on Rdio Scanner.

My RTLSDR Airband works in scan mode and preparing me a files with single transmision to Dirwatch folder with filename NAME_20220127_160810_145787500.mp3 with mask NAME#DATE_#TIME_#HZ. Files are load to Rdio but I can not find any option to prepare separate Talkgroups for files with mask frequency in filename in #HZ. 

Is it any option to turn on scanner on my RTLSDR for 5 frequences that will be 5 others talkrooms in Rdio?

I would like to plug in four RTLSDR dongles to my raspberry for scanning 20 frequencies in one second and save files to Rdio.

Help me please!

**Comment by chuot** [comment]
Honestly, I've never tried in scan mode because that kinda defeats the purpose of Rdio Scanner, where you shouldn't miss anything.

This is why I did all my tests in multichannel mode. I use the `filename_template` property to indicate the talkgroupId. For example with `filename_template` set to `cymx_119100` I get filenames like `cymx_119100_20220127_140353_119100000` which with a mask like `cymx_#TG_#DATE_#TIME_#HZ` I can achieve what you want.

However, I understand that sometimes scan mode is necessary. For this reason, I will add on the next version a new mask #TGHZ which will set both the talkgroupId and the frequency to the same value. That should do the trick, but until then you're out of luck in scan mode.

**Reply by poncheck** [reply]
Thank for your help! Everything working very well!

**Reply by samsonpwnz** [reply]
Can some one post a config or maybe some screenshots on how to get this to work. I would like to use multichannel to record 7-8 vhf-air freqs and then have them all feed into rdio-scanner. I guess the part I'm stuck at is with the mask and TG settings. Thanks.

**Comment by chuot** [comment]
@samsonpwnz There's already an example in the [docs/examples/rtlsdr-airband](https://github.com/chuot/rdio-scanner/tree/master/docs/examples/rtlsdr-airband) folder.

**Reply by samsonpwnz** [reply]
I should of dug around a bit more before asking the question, thanks for pointing this out. I am itching to get this to work!
